plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/persistence/SerializableLibraryClasspathContainer.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.google.cloud.tools.eclipse.appengine.libraries.persistence;
+
+import com.google.cloud.tools.eclipse.appengine.libraries.LibraryClasspathContainer;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.core.IClasspathEntry;
+
+public class SerializableLibraryClasspathContainer {
+
+  private String description;
+  private String path;
+  private SerializableClasspathEntry[] entries;
```

Comment:
Any chance this is going to get hit by multiple threads? It doesn't look thread-safe.